operator/config/manifests/bases/odigos-operator.clusterserviceversion.yaml

```diff
@@ -16,6 +16,10 @@ metadata:
     features.operators.openshift.io/token-auth-aws: "false"
     features.operators.openshift.io/token-auth-azure: "false"
     features.operators.openshift.io/token-auth-gcp: "false"
+    operatorframework.io/suggested-namespace: "odigos-operator-system"
```

Comment:
why not `odigos-system`?
this is the same namespace where odigos itself will be deployed, right?

Reply:
`x-operator-system` is common for operators, and in this case the operator is installed in the namespace before Odigos is installed and managed by the operator